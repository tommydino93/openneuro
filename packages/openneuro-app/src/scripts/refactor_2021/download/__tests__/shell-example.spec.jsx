import React from 'react'
import { shallow } from 'enzyme'
import ShellExample from '../shell-example.jsx'

describe('dataset/download/ShellExample', () => {
  it('renders successfully', () => {
    const wrapper = shallow(<ShellExample />)
    expect(wrapper).toMatchSnapshot()
  })
})
