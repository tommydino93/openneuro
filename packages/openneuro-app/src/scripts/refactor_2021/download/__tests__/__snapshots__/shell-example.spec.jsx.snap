// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dataset/download/ShellExample renders successfully 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
